src/main.tsx

```diff
@@ -15,10 +15,14 @@ const prepare = async (): Promise<void> => {
 
 prepare().then(() => {
   useI18next();
-  render(
+
+  const container = document.getElementById("root");
+  if (!container) throw new Error("container element is missing.");
+
+  const root = createRoot(container);
```

Comment:
I Changed to use `createRoot` according to [documentation](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis).

Reply:
I think we can use `createRoot(container!)` so we don't need to throw Error when `container` is not existed
<img width="923" alt="image" src="https://user-images.githubusercontent.com/31472539/171126633-f188fcb6-f1a9-47b5-b676-2f04dbef0256.png">

Reply:
@mvn-binhnguyen-hn We have accepted your suggestion! [dd956a3](https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/28/commits/dd956a31d2be2d04a4e658bf86edbc6cd04dd78f)